standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/cache/TestCachedStore.java

```diff
@@ -1566,160 +1643,6 @@ private Table createUnpartitionedTableObject(Database db) {
     return tbl;
   }
 
-  private TableAndColStats createUnpartitionedTableObjectWithColStats(Database db) {
```

Comment:
Why is this test case being removed ?

Reply:
This is not a test case it is just a private unused method.

Reply:
Ok, thanks for the cleanup.